internal/broker/broker.go

```diff
@@ -49,50 +49,53 @@ type Mqconf struct {
 
 // NewMQ creates a new Broker that can communicate with a backend
 // amqp server.
-func NewMQ(c Mqconf) *AMQPBroker {
-	brokerURI := buildMqURI(c.Host, c.User, c.Password, c.Vhost, c.Port, c.Ssl)
+func NewMQ(configuration MQConf) (*AMQPBroker, error) {
+	brokerURI := buildMQURI(configuration.Host, configuration.User, configuration.Password, configuration.Vhost, configuration.Port, configuration.Ssl)
```

Comment:
What's with these java style overly verbose naming schemas?

Reply:
It's not Java :) I just like when meeting some variable you can easily understand what it is just by looking at its name, instead of guessing what `a`, `b` and `c` can be :)

Reply:
👯